introduction.md

```diff
@@ -1 +1,5 @@
 Hello I'm anonimus. My favorite path is Ruby.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?